account/templates/registration/password_reset_subject.txt

```diff
@@ -0,0 +1 @@
+AF password reset
```

Comment:
what is the use of this?

Reply:
Its not being used in development environment.

Reply:
so it will be of use during production release?